src/TableV8/TableFilters/index.tsx

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
moved filter components from platform-UI to uikit so they can be reused

Reply:
Tested + working with local Summary table